wasm/module_test.go

```diff
@@ -14,21 +14,29 @@ import (
 )
 
 func TestReadModule(t *testing.T) {
-	fnames, err := filepath.Glob("testdata/*wasm")
+	fnames, err := filepath.Glob(filepath.Join("testdata", "*.wasm"))
 	if err != nil {
 		t.Fatal(err)
 	}
 	for _, fname := range fnames {
-		raw, err := ioutil.ReadFile(fname)
-		if err != nil {
-			t.Error(err)
-			continue
-		}
-		r := bytes.NewReader(raw)
-		_, err = wasm.ReadModule(r, nil)
-		if err != nil {
-			t.Errorf("error reading module %q: %v", fname, err)
-			continue
-		}
+		testModule(t, fname)
```

Comment:
What's the reason you chose to to factor all this out into `testModule`? It doesn't seem to be helpful. You're not making use of any `defer`s. All `return`s can just be replaced with `continue`s and it'd be just fine inside the `TestReadModule` for loop. I don't think it helps, it only hurts readability IMO. But it's not a huge deal.